recipes-ni/ni-shutdown-guard/ni-shutdown-guard_1.0.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Shutdown/reboot guard utility for NILRT"
+DESCRIPTION = "Utility to prevent shutdown/reboot to protect critical operations"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+SRC_URI = "\
+	file://rguard \
+	file://holdoff-shutdown \
+"
+
+FILES_${PN} += "\
+	${sbindir}/rguard \
+	${sysconfdir}/init.d/holdoff-shutdown \
+"
+
+DEPENDS += "shadow-native pseudo-native update-rc.d-native"
```

Comment:
Why do you need `shadow-native` and `pseudo-native` here?

Reply:
Assumed they were needed, but doesn't seem to be the case. Removed